docs/functions/data.rst

```diff
@@ -140,6 +140,31 @@ Development Environment
    .. versionadded:: 2.1
 
 
+Disks and Memory
+----------------
+
+.. function:: _lp_disk -> var:lp_disk, var:lp_disk_human, var:lp_disk_perc
+
+   Gather information about the current state of the hard drive hosting the
+   *current directory*:
+
+   * occupied space in kilo-bytes (``lp_disk``),
```

Comment:
Wouldn't hurt to be specific that this is 1024 byte KBs, not the 1000 byte version.